packages/@ember/-internals/glimmer/tests/integration/helpers/unique-id-test.js

```diff
@@ -102,6 +102,24 @@ if (EMBER_UNIQUE_ID_HELPER) {
         });
       }
 
+      ['@test it only generates valid selectors']() {
+        let iterations = 1000;
+        let reNumericStart = /^\d/;
+
+        let template = '<p>{{unique-id}}</p>'.repeat(iterations);
+        super.render(template);
+
+        for (let i = 0; i < iterations; i++) {
+          let textNode = this.nthChild(i).firstChild;
+          let text = textNode.data;
+
+          this.assert.false(
+            reNumericStart.test(text),
+            `{{unique-id}} should produce valid selectors` + text
+          );
+        }
```

Comment:
Ah. I mean that we should _invoke_ the `{{unique-id}}` helper 1000 times in the test (since it includes some randomness, the iterations increase the chances of actually catching the bugs/confirming the behavior). 

The easiest way to do it is probably generate a template with 1000 invocations, each surrounded by some kind of marker (eg `[{{unique-id}}] [{{unique-id}}] …`) then we can assert that each of the 1000 instances (which would be all different) all are valid selectors. Does that make sense?

Reply:
> The easiest way to do it is probably generate a template with 1000 invocations, each surrounded by some kind of marker (eg [{{unique-id}}] [{{unique-id}}] …) then we can assert that each of the 1000 instances (which would be all different) all are valid selectors. Does that make sense?

I might be misunderstanding but isn't that pretty much exactly what I did here? 😅

Reply:
You’re right, not sure what I read 😅 can you fix the lint? Then I think we are good to go